modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -21,6 +21,13 @@ export function isScriptType2Of3(t: string): t is ScriptType2Of3 {
 
 export type ScriptType = ScriptTypeP2shP2pk | ScriptType2Of3;
 
+/**
+ * @return true iff scriptType requires witness data
+ */
+export function hasWitnessData(scriptType: ScriptType): scriptType is 'p2shP2wsh' | 'p2wsh' | 'p2tr' {
+  return ['p2shP2wsh', 'p2wsh', 'p2tr'].includes(scriptType);
```

Comment:
I kinda love cases like this where a maximum of 3 3-character comparisons are required to search in an array, so a hashed or tree-type data structure can't beat a linear array search.